src/store/modules/session.js

```diff
@@ -22,9 +22,10 @@ const state = () => ({
   nomination: false,
   votes: [],
   lockedVote: 0,
-  votingSpeed: 3000,
+  votingSpeed: 1000,
```

Comment:
1s is way too fast for smaller groups, which happen more often than 12+ player groups. Please keep it at 3 by default. If it really bothers you, I'd suggest adding this to the persistence.js to store the value in the browser. ;-)

Reply:
I've not seen a game use anything slower than 1s, but it doesn't really bother me (I tend not to ST).